### ISSUE TITLE
can not compile with 'USE_ICEPACK' set to 'ON'

### ISSUE THREAD

**Issue by hegish** [FESOM/fesom2#137]
with gfortran 10.2.0 anc cmake 3.20.3 I get this error:

[ 52%] Building Fortran object src/CMakeFiles/fesom.dir/icepack_drivers/icedrv_kinds.F90.o
.../src/icepack_drivers/icedrv_kinds.F90:11:11:

   11 |       use icepack_intfc, only: char_len       => icepack_char_len
      |           1
Fatal Error: Cannot open module file 'icepack_intfc.mod' for reading at (1): No such file or directory
compilation terminated.

**Comment by lzampier** [comment]
Hi @hegish, have you downloaded the icepack code first, as described in the [documentation](https://fesom2.readthedocs.io/en/latest/icepack_in_fesom.html#compilation)?

**Comment by lzampier** [comment]
Hi @hegish, have you managed to compile with icepack? Can we close this issue?

**Comment by hegish** [comment]
I will try it again this week. You are probably right, I have only the icepack files included with Fesom. If your suggestion is true though, we should change the cmake configuration phase so it leads to an error if the external icepack data is missing. Should I do that?

**Comment by lzampier** [comment]
That's a very good idea! Is it possible to make cmake download the missing files from the repository if they are not there? I am not a cmake expert so I am not sure about that... Otherwise, the error message would be a good improvement.

**Comment by koldunovn** [comment]
To be hones I would just do the warning and point user to documentation instead of downloading files automatically.

**Comment by hegish** [comment]
Yes, downloading would be possible via cmake. And yes, I also do not recommend it in this case.  (:
